package io.qiot.covid19.edge.util.exception;